src/lib/mdns/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -544,34 +560,28 @@ CHIP_ERROR AdvertiserMinMdns::Advertise(const CommissionAdvertisingParameters &
 
 FullQName AdvertiserMinMdns::GetCommisioningTextEntries(const CommissionAdvertisingParameters & params)
 {
+    char * txtFields[9];
```

Comment:
It would be good if we had some sort of checks below that `numTxtFields` is never bigger than `ArraySize(txtFields)` when we write to `txtFields`...

Reply:
It's sized to be the max number of text fields we're allowed to add per the spec, but that is completely not obvious from the way I've written it. If I change that to kMaxTxtFields, would that be OK?

Reply:
With a comment about how that value comes from the spec, that would be fine, yes.

Reply:
Perfect - added in the next commit. Thanks for pointing that out - much more readable now.